azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -173,6 +181,16 @@ public FlowRunnerManager(final Props props,
       this.projectDirectory.mkdirs();
     }
 
+    this.projectDirMaxSizeInMB = props.getLong(ConfigurationKeys.PROJECT_DIR_MAX_SIZE,
+        2000000); // default value as 2TB
```

Comment:
Why is 2 TB a good default? I don't think most of the deployment will require such a big cache.

Reply:
updated in #1812